[opdocs] Kommander: Split clusters into multiple pages (#3400)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/clusters/attach-cluster/index.md

## Attach Kubernetes Cluster

> You can attach an existing cluster directly to Kommander. At attachment time certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure the attached cluster has sufficient resources. For more information on platform services and customizing them, see [workspace platform services][workspace_platform_services].

Author: Would this and https://github.com/mesosphere/dcos-docs-site/pull/3400/commits/0eb9bafa1840de5cf5919e54569654634b702d58 suffice for https://jira.d2iq.com/browse/COPS-6437 ? I didn't think we needed to include the specifics here as it may be too much detail and also subject to change (e.g. the namespaces that are created)

Reviewer: I think that would work!

> You can attach an existing cluster directly to Kommander. Upon attachment, certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure that the attached cluster has sufficient resources. For more information on platform services and how to customize them, see [workspace platform services][workspace_platform_services].

Reviewer: ```suggestion
You can attach an existing cluster directly to Kommander. At attachment time certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure the attached cluster has sufficient resources. For more information on platform services and customizing them, see [workspace platform services][workspace_platform_services].
```

Selecting the **Attach Cluster** option displays the **Connection Information** dialog box. This dialog box accepts a kubeconfig file, that you can paste, or upload into the field. In the **Context** select list, you can select the intended context or change the display name provided with the config. You can add labels to classify your cluster and select the platform services to install.

> Platform services extend the functionality of Kubernetes and allow you to deploy ready-to-use logging and monitoring stacks by federating platform services when attaching a cluster to Kommander. For more information, read our documentation about [workspace platform services][workspace_platform_services].

Reviewer: ```suggestion
Platform services extend the functionality of Kubernetes and allow you to deploy ready-to-use logging and monitoring stacks by federating platform services when attaching a cluster to Kommander. For more information, refer to [workspace platform services][workspace_platform_services].
```
